source/_dashboards/conditional.markdown

```diff
@@ -39,7 +44,12 @@ card:
 
 *one is required (`state` or `state_not`)
 
-Note: Conditions with more than one entity are treated as an 'and' condition. This means that for the card to show, *all* entities must meet the state requirements set.
+## Link between conditions
+
+Conditions can be either linked with `and` or with `or`. This can be set via the `link_conditions` configuration.
+
+- If the conditions are linked with `and`, then **all** entities must meet the state requirement.
+- If the conditions are linked with `or`, then only **one** entity of all entities must meet the state requirement.
 
 ## Examples
```

Comment:
I think that it would be good to add it in the example as well : 

```yaml
type: conditional
link_conditions: "and"  # << 
conditions:
  - entity: light.bed_light
    state: "on"
  - entity: switch.decorative_lights
    state_not: "off"
card:
  type: entities
  entities:
    - device_tracker.demo_paulus
    - cover.kitchen_window
    - group.kitchen
    - lock.kitchen_door
    - light.bed_light
```